modelmapper/stats.py

```diff
@@ -473,14 +473,23 @@ def inspect_item(self, field_name, item):
         :param settings:
             Settings object
         """
+        item = item.strip()
         results = []
+        already_matched = False
         for matcher in self.matchers:
-            if not matcher.match(item):
+            # if isinstance(matcher, (PositiveIntMatcher, StringMatcher)):
+            #     import pytest; pytest.set_trace()
+            #     print(matcher)
+            if already_matched and isinstance(matcher, StringMatcher):
```

Comment:
I'm not sure I understand this change

Reply:
I guess I'm wondering if we need to name the matchers and explicitly say which ones conflict instead of calling the `is_exclusive()` method. We could call this new method `conflicts_with()` and drop any remaining conflicting matchers?